tests/Config.test.ts

```diff
@@ -44,7 +44,7 @@ describe("Test of Config", () => {
         assert.strictEqual(config.database.database, "stoa");
         assert.strictEqual(config.database.port.toString(), "3306");
         assert.strictEqual(config.database.password.toString(), "12345678");
-        assert.strictEqual(config.logging.folder, "/stoa/logs");
+        assert.notStrictEqual(config.logging.folder, null);
```

Comment:
Isn't there a path package that would abstract this away ?

Reply:
I tried, but I couldn't find the package.

Reply:
```suggestion
        assert.strictEqual(config.logging.folder, path.resolve(sdk.getInitCWD(), "/stoa/logs"));
```

Reply:
@MichaelKim20 Thank you. I updated it.